web-client/public/js/graph.js

```diff
@@ -1217,35 +1236,44 @@ export var drawGraph = function (network, sliderController, nodeColoring) {
         return name1 > name2 ? 1 : -1;
     };
 
-    let layout = false;
+    gridLayout = false;
 
     var GRID_LAYOUT_BUTTON = "#gridLayoutButton";
     $(GRID_LAYOUT_BUTTON)[0].value = "Grid Layout";
-    $(GRID_LAYOUT_BUTTON).on("click", {handler: this}, function (event) { // eslint-disable-line no-unused-vars
+    $(GRID_LAYOUT_BUTTON).on("click", {handler: this},
```

Comment:
@dondi I was working on moving this force graph layout controller to upload.js, just like the gray edge, dashed line, and physics option controllers are. However, I ran into a problem where the container decides to not load if I change 

`$(GRID_LAYOUT_BUTTON).on("click", {handler: this}, function (event) { ... }` 

to

`$(GRID_LAYOUT_BUTTON).on("click", {handler: this}, gridLayoutToggle)
 var  gridLayoutToggle = function (event) { ... }`

Reply:
Timeout timeout—I thought the plan was to move _one_ thing, _very focused_, into the new cycle first? This is a huge set of changes that seems to be disproportionate to moving that single value. I think I need some clarification on how this task, which was supposed to be pretty small and limited, has become this gigantic diff.

Reply:
@dondi so this isn't actually moving anything into the new set of files that I have created, this is organizing the existing codebase before tackling each bit step by step. I realized the need for this when I started moving bits and pieces and realized they were scattered all over graph.js

Reply:
OK, so this is a distinct task, and not the original MVC refactor. That's fine, but I thought this reorganization would _follow_ the establishment of the clean setupHandlers -> grnState -> updateApp cycle. Don't we want to establish that as a solid foundation first, before then embarking on reorganizing the code?

Reorganizing the code now seems like potential duplicate work, because until you have the clean MVC loop, it won't really be clear where the new code will go.